<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 170, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_basico">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(91, 140, 255);
background-color: rgb(0, 0, 255);
border: 5px solid black;
border-radius: 25px;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Básico</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_op_defecto">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(91, 140, 255);
background-color: rgb(0, 0, 255);
border: 5px solid black;
border-radius: 25px;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Pre-Selección</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_texto_detallado">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(91, 140, 255);
background-color: rgb(0, 0, 255);
border: 5px solid black;
border-radius: 25px;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Texto Descriptivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_informacion">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(91, 140, 255);
background-color: rgb(0, 0, 255);
border: 5px solid black;
border-radius: 25px;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Información</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pregunta">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(91, 140, 255);
background-color: rgb(0, 0, 255);
border: 5px solid black;
border-radius: 25px;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Pregunta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_advertencia">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(91, 140, 255);
background-color: rgb(0, 0, 255);
border: 5px solid black;
border-radius: 25px;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Advertencia</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_critico">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(91, 140, 255);
background-color: rgb(0, 0, 255);
border: 5px solid black;
border-radius: 25px;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Crítico</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_botones_personalizados">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>530</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(91, 140, 255);
background-color: rgb(0, 0, 255);
border: 5px solid black;
border-radius: 25px;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Botones Personalizados</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Imagenes/recursos1.qrc">:/logos/LogoUAT.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>25</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Imagenes/recursos1.qrc">:/logos/LogoFIT.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>520</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Imagenes/recursos1.qrc">:/logos/LogoEquipo.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Imagenes/recursos1.qrc"/>
 </resources>
 <connections/>
</ui>
